site/it/about/bib.md

```diff
@@ -0,0 +1,131 @@
+
+<pre>
+@misc{tensorflow2015-whitepaper,
+title={ {TensorFlow}: Large-Scale Machine Learning on Heterogeneous Systems},
+url={https://www.tensorflow.org/},
+note={Software available from tensorflow.org},
+author={
+    Mart\'{\i}n~Abadi and
+    Ashish~Agarwal and
+    Paul~Barham and
+    Eugene~Brevdo and
+    Zhifeng~Chen and
+    Craig~Citro and
+    Greg~S.~Corrado and
+    Andy~Davis and
+    Jeffrey~Dean and
+    Matthieu~Devin and
+    Sanjay~Ghemawat and
+    Ian~Goodfellow and
+    Andrew~Harp and
+    Geoffrey~Irving and
+    Michael~Isard and
+    Yangqing Jia and
+    Rafal~Jozefowicz and
+    Lukasz~Kaiser and
+    Manjunath~Kudlur and
+    Josh~Levenberg and
+    Dandelion~Man\'{e} and
+    Rajat~Monga and
+    Sherry~Moore and
+    Derek~Murray and
+    Chris~Olah and
+    Mike~Schuster and
+    Jonathon~Shlens and
+    Benoit~Steiner and
+    Ilya~Sutskever and
+    Kunal~Talwar and
+    Paul~Tucker and
+    Vincent~Vanhoucke and
+    Vijay~Vasudevan and
+    Fernanda~Vi\'{e}gas and
+    Oriol~Vinyals and
+    Pete~Warden and
+    Martin~Wattenberg and
+    Martin~Wicke and
+    Yuan~Yu and
+    Xiaoqiang~Zheng},
+  year={2015},
+}
+</pre>
+
+O in forma testuale:
+
+<pre>
+Martín Abadi, Ashish Agarwal, Paul Barham, Eugene Brevdo,
+Zhifeng Chen, Craig Citro, Greg S. Corrado, Andy Davis,
+Jeffrey Dean, Matthieu Devin, Sanjay Ghemawat, Ian Goodfellow,
+Andrew Harp, Geoffrey Irving, Michael Isard, Rafal Jozefowicz, Yangqing Jia,
+Lukasz Kaiser, Manjunath Kudlur, Josh Levenberg, Dan Mané, Mike Schuster,
+Rajat Monga, Sherry Moore, Derek Murray, Chris Olah, Jonathon Shlens,
+Benoit Steiner, Ilya Sutskever, Kunal Talwar, Paul Tucker,
+Vincent Vanhoucke, Vijay Vasudevan, Fernanda Viégas,
+Oriol Vinyals, Pete Warden, Martin Wattenberg, Martin Wicke,
+Yuan Yu, and Xiaoqiang Zheng.
+TensorFlow: Large-scale machine learning on heterogeneous systems,
+2015. Software available from tensorflow.org.
+</pre>
+
+
+
+## TensorFlow: Un sistema per Machine Learning su Larga Scala
+
+[Accedi a questo white paper.](https://www.usenix.org/system/files/conference/osdi16/osdi16-abadi.pdf)
+
+**Sommario:** TensorFlow è un sistema di machine learning che
+opera su larga scala ed in ambienti eterogenei. TensorFlow
+usa grafi dataflow per rappresentare la computazione,
+stati condivisi, e le operazioni che cambiano questi stati.
+Esso mappa i nodi di un grafo dataflow tra molte macchine in cluster, 
+e tra macchine appartenenti a diversi dispositivi di computazione
+inclusi CPU multicore, GPUs generiche, e ASICs progettati in proprio
+detti Tensor Processing Units (TPUs). Quest'architettura fornisce
+flessibilità allo sviluppatore di applicazioni: mentre nei precedenti 
+progetti basati su “server parametrici” la gestione degli stati condivisi
+è realizzata all'interno del sistema, TensorFlow permette agli 
+sviluppatori di sperimentare con nuove ottimizzazioni e algoritmi di addestramento.
+TensorFlow supporta varie applicazioni, con un particolare focus
```

Comment:
"con un particolare focus": looking at the English text I would just say "focalizzando" here.

Reply:
I agree